clang/lib/Serialization/ASTWriterDecl.cpp

```diff
@@ -1376,6 +1377,16 @@ void ASTDeclWriter::VisitBlockDecl(BlockDecl *D) {
   Code = serialization::DECL_BLOCK;
 }
 
+void ASTDeclWriter::VisitOutlinedFunctionDecl(OutlinedFunctionDecl *D) {
+  Record.push_back(D->getNumParams());
+  VisitDecl(D);
+  Record.push_back(D->isNothrow() ? 1 : 0);
```

Comment:
Slight preference to move this 'down' and put the params and the num-params next to eachother.

Reply:
Done.